ruby/vampires.rb

```diff
@@ -0,0 +1,126 @@
+vamp_con1 = 'Definitely a vampire!'
+
+vamp_con2 = '...Probably not a vampire.'
+
+vamp_con3 = 'Probably a vampire.'
+
+vamp_con4 = 'Almost certainly a vampire.'
+
+vamp_con5 = 'Results inconclusive.'
+
+
+puts 'How many employees will be processed?'
+
+employees_to_process = gets.chomp
+while employees_to_process.to_i > 0
+
+	puts 'What is your name?'
+	name = gets.chomp
+
+	if name.downcase == 'tu fang' || name.downcase == 'drake cula' || name.downcase == 'dracula' || name.downcase == 'nosferatu'
```

Comment:
I think this should be your first check. If their name is Drac or Fang then we know we have a vampire. You can nest the rest of the checks under an else following this check. if vamp == true then no reason to check anything else.

The below is an example of handling the control flow in a more efficient computational pattern. Also, it is important to practice nesting conditions as many control flows will require you to do so.

```
if name == vamp
      vampire = true
else
      if other conditions
      elsif other conditions
      end
end
```